games-puzzle/kpat/kpat-17.12.1.recipe

```diff
@@ -0,0 +1,145 @@
+CHECKSUM_SHA256="5241e8092bf4820b4ad2a016fed0928ed12815edb050dbb3f5512103aca0f293"
+#PATCHES="kpat-$portVersion.patchset"
+ADDITIONAL_FILES="kpat.rdef.in"
+	
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	kpat$secondaryArchSuffix = $portVersion
+	app:kpat$secondaryArchSuffix
+	lib:libkcardgame$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:dbus_launch
+	lib:libKF5Attica$secondaryArchSuffix
+	lib:libKF5Auth$secondaryArchSuffix
+	lib:libKF5Bookmarks$secondaryArchSuffix
+	lib:libKF5Codecs$secondaryArchSuffix
+	lib:libKF5Completion$secondaryArchSuffix
+	lib:libKF5ConfigCore$secondaryArchSuffix
+	lib:libKF5ConfigWidgets$secondaryArchSuffix
+	lib:libKF5CoreAddons$secondaryArchSuffix
+	lib:libKF5Crash$secondaryArchSuffix
+	lib:libKF5DBusAddons$secondaryArchSuffix
+	lib:libKF5GuiAddons$secondaryArchSuffix
+	lib:libKF5I18n$secondaryArchSuffix
+	lib:libKF5ItemViews$secondaryArchSuffix
+	lib:libKF5JobWidgets$secondaryArchSuffix
+	lib:libKF5KDEGames$secondaryArchSuffix
+	lib:libKF5KIOCore$secondaryArchSuffix
+	lib:libKF5NewStuff$secondaryArchSuffix
+	lib:libKF5Service$secondaryArchSuffix
+	lib:libKF5WidgetsAddons$secondaryArchSuffix
+	lib:libKF5XmlGui$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Qml$secondaryArchSuffix
+	lib:libQt5Quick$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	extra_cmake_modules >= 5.39
+	haiku${secondaryArchSuffix}_devel
+	devel:libKF5Attica$secondaryArchSuffix
+	devel:libKF5Auth$secondaryArchSuffix
+	devel:libKF5Bookmarks$secondaryArchSuffix
+	devel:libKF5Codecs$secondaryArchSuffix
+	devel:libKF5Completion$secondaryArchSuffix
+	devel:libKF5ConfigCore$secondaryArchSuffix
+	devel:libKF5ConfigWidgets$secondaryArchSuffix
+	devel:libKF5CoreAddons$secondaryArchSuffix
+	devel:libKF5Crash$secondaryArchSuffix
+	devel:libKF5DBusAddons$secondaryArchSuffix
+	devel:libKF5GuiAddons$secondaryArchSuffix
+	devel:libKF5I18n$secondaryArchSuffix
+	devel:libKF5ItemViews$secondaryArchSuffix
+	devel:libKF5JobWidgets$secondaryArchSuffix
+	devel:libKF5KDEGames$secondaryArchSuffix
+	devel:libKF5KIOCore$secondaryArchSuffix
+	devel:libKF5NewStuff$secondaryArchSuffix
+	devel:libKF5Service$secondaryArchSuffix
+	devel:libKF5Solid$secondaryArchSuffix
+	devel:libKF5WidgetsAddons$secondaryArchSuffix
+	devel:libKF5XmlGui$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Qml$secondaryArchSuffix
+	devel:libQt5Quick$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:gettext$secondaryArchSuffix
+	cmd:make
+	cmd:python2
+	cmd:update_mime_database$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	# disable docs generation until libxml package is fixed
+	sed -i s/DocTools/#DocTools/g CMakeLists.txt
+	sed -i s/add_subdirectory\(doc\)/#add_subdirectory\(doc\)/g CMakeLists.txt
```

Comment:
Please use quotes instead. Also, this style is used more often in HaikuPorts.

Please consider moving this to the `PATCH()` function.

```diff
-	sed -i s/DocTools/#DocTools/g CMakeLists.txt
-	sed -i s/add_subdirectory\(doc\)/#add_subdirectory\(doc\)/g CMakeLists.txt
+	sed -e 's/DocTools/#DocTools/g' \
+		-e 's/add_subdirectory(doc)/#add_subdirectory(doc)/g' \
+		-i CMakeLists.txt

Reply:
Well, i've seen so many recipes doing so, i just followed the same pattern.

Reply:
@fbrosson Can you give us some metrics?

Reply:
(Sorry for this late reply.)
I think most of our recipes use double quotes. We probably also have some with single quotes.
I like to use double quotes because it's more eye-friendly.
I can/will do some metrics, but a bit later. (There are lots of PRs waiting to be reviewed or merged...)